src/module-1/styles/scss/components/_product.scss

```
@@ -0,0 +1,142 @@
+.product {
+  align-items: center;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px #00000022;
+  display: flex;
+  flex-direction: column;
+  min-height: 486px;
+  width: 400px;
+  margin: 30px;
+
+  &__img {
+    height: 272px;
+    margin-top: 31px;
+    max-width: 100%;
+    padding: 15px;
+  }
+
+  &__price {
+    align-self: flex-end;
+    margin-right: 20.49px;
+    margin-left: 247px;
+    margin-top: 36px;
+
+    color: $heavy-metal;
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  &__title {
+    align-self: flex-start;
+    line-height: 19px;
+    margin-left: 23.72px;
+    margin-top: 25px;
+    min-height: 19px;
+
+    color: $heavy-metal;
+    font-size: 16px;
+    font-weight: 400;
+  }
+
+  &__description {
+    line-height: 14px;
+    margin-right: 38px;
+    margin-top: 3px;
+    min-height: 14px;
+    width: 315px;
+
+    color: $heavy-metal;
+    font-size: 12px;
+    font-weight: 300;
+  }
+
+  &__btn {
+    align-items: flex-start;
+    display: flex;
+    justify-content: center;
+    min-width: 200px;
+    margin-top: 22px;
+
+    &--heart {
+      background: url("../../img/icons/heart-1@2x.svg") no-repeat;
```

```suggestion
      background: url("../../img/icons/heart.svg") no-repeat;
```  
consider using simplified names for your local images